<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settime</class>
 <widget class="QDialog" name="Settime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Resources/time.PNG);</string>
  </property>
  <widget class="QPushButton" name="SetOk">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TimeEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入同步的时间：</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LocalTime">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
